paper/config/plugins/SimpleAdminHacks/config.yml

```diff
@@ -454,20 +454,29 @@ hacks:
-       - "Build a mighty fortress or anime pixel art! Type \"/ctdl\" to see a list of other reinforcement types. Learn about the Bastion plugin if your monastery requires a moat."
-       - "   &1JukeAlert&r\nAn alarm system for your castle and a way to spy on your friends and enemies. Use Citadel to reinforce a noteblock or a jukebox. Noteblocks will alert you when someone not on the group walks past, and jukeboxes will store a log of nearby activities"
-       - "such as opening chests and breaking blocks. \"/ja\" will open a GUI. It is dangerous to steal from people who have mastered both JukeAlert and PrisonPearl..."
-       - "   &1FactoryMod&r\nStoke the fires of industry by filling a chest with 8x64 charcoal, placing a crafting table next to the chest, and then a furnace containing more charcoal next to the crafting table. Hit the crafting table with a stick twice, add wood to the chest, and then"
-       - "hit the furnace with a stick. You are now a wealthy charcoal baron capable of producing one of many items faster and more efficiently than those savages still using furnaces! Type \"/fm\" for more info – almost all items have factories."
-       - "   &1RealisticBiomes&r\nYou will starve! Crops and trees require sunlight or glowstone, grow slowly across the board, and do not grow in inappropriate biomes. This encourages trade and warfare because diverse crops are required for FactoryMod XP production."
-       - "   &5Closing Remarks&r\nType /help for more info on everything. Go to https://redd.it/j3nk2f for a more detailed guide. There is much more to learn."
-       - "If you would like to report hacks or glitches, please message the admins through subreddit modmail at reddit.com/r/CivMC\n\nDisclaimer: &2TangentialThreat&r is not staff and should not be trusted."
+       - "&5Hello and welcome to CivMC, &2${player}!\n\n   A server based on nation building and player freedom. Where admin control is minimal, and players dispense their own justice." 
+	   	   - "   &1Namelayer&r\nPocket radios! Create or join chat groups to communicate with friends all over CivMC. By default you have joined global chat or ! chat. You can switch from local chat (1000 block radius) to this one by performing the command: /g !"
+	   - "Go back to localchat by performing the /g command on it own. This works for any group you are allowed to speak in. just switch ! for the name of the group. \nCreate your own group using the GUI by perfoming the /nl Command."
+	  	   - "   &1Citadel&r\nAnother use for your Namelayer groups! Slow down raiders by reinforcing blocks such as your house or your chests and doors to lock them. Reinforced blocks take multiple breaks to drop for players not on the namelayer group."
+	   - "To reinforce a block perform the command /ctr (Groupname) while holding stone, iron or diamond. then punch any block to reinforce it to that group. Now only your allies have ease of access! Tip: Use /ctf instead to reinforce while building!"
```

Comment:
- `(/ctr <groupname>)`
- Capitalize 'Then
- 'have ease of access' -> can interact with that block.